### DISCUSSION TITLE
Whitelisting new contributors for Action approval

### DISCUSSION THREAD

**Discussion by martinbonnin** [community/community#31583] [Actions]
New contributors to a repo need to have their workflow approved before they are run ([GitHub doc](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-required-approval-for-workflows-from-public-forks)). This is nice and prevents a rogue user to extract secrets or potentially run miners or other from pull requests.

The problem happens when a first time contributors edits their pull request. The actions has to be re-approved every time.

Is it possible to whitelist the new contributor once trust has been established so that workflows are always run subsequently?

**Comment by jfullstackdev** [comment]
is that repo in a personal account ? how about if it's in Org ? will adding the user as member resolve the issue ?

**Reply by martinbonnin** [reply]
> is that repo in a personal account ? how about if it's in Org ? 

It's an org repo (it's https://github.com/apollographql/)

> will adding the user as member resolve the issue ?

Not really. We don't want to add every member that opens a pull request. Ideally a checkbox that says "do not ask again" would be 👌
